webapi/Models/Response/ProposedPlan.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'm thinking we could delete this file if we decided we don't need planning other than that provided by function calling